Classes/ViewHelpers/Widget/Controller/PaginateController.php

```diff
@@ -137,8 +137,28 @@ public function indexAction($currentPage = 1)
         }
 
         if ($this->currentPage > 1) {
-            $pageLabel = \TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate('paginate_overall', 'news', [$this->currentPage, $this->numberOfPages]);
-            $GLOBALS['TSFE']->page['title'] = $GLOBALS['TSFE']->page['title'] . ' - ' . trim($pageLabel, '.');
+            $pageLabel = \TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate(
+                'paginate_overall',
+                'news',
+                [
+                    $this->currentPage,
```

Comment:
doesn't that require a new label if the arguments change?

Reply:
I'm not sure what label do you mean. Do you mean a new language label for the translation?
This code for `$pageLabel` was just formatted to shorten the overall line length. The functionality should be the same as before.